fk/lisjp/src/main/java/se/inera/intyg/common/lisjp/v1/model/converter/InternalToCertificate.java

```diff
@@ -681,4 +772,866 @@ private static CertificateDataValueDiagnosis createDiagnosis(String id, Diagnos
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.SOKA_NYTT_ARBETE.getId()),
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.BESOK_PA_ARBETSPLATSEN.getId()),
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.ERGONOMISK_BEDOMNING.getId()),
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.HJALPMEDEL.getId()),
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.KONFLIKTHANTERING.getId()),
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.KONTAKT_MED_FORETAGSHALSOVARD.getId()),
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.OMFORDELNING_AV_ARBETSUPPGIFTER.getId()),
+                                singleExpression(ArbetslivsinriktadeAtgarderVal.OVRIGT.getId())
+                            )
+                        )
+                        .id(Arrays.asList(ArbetslivsinriktadeAtgarderVal.INTE_AKTUELLT.getId()))
+                        .build(),
+                    CertificateDataValidationDisable.builder()
+                        .questionId(ARBETSLIVSINRIKTADE_ATGARDER_SVAR_ID_40)
+                        .expression(
+                            singleExpression(ArbetslivsinriktadeAtgarderVal.INTE_AKTUELLT.getId())
+                            )
+                        .id(
+                            Arrays.asList(
+                                ArbetslivsinriktadeAtgarderVal.ARBETSTRANING.getId(),
+                                ArbetslivsinriktadeAtgarderVal.ARBETSANPASSNING.getId(),
+                                ArbetslivsinriktadeAtgarderVal.SOKA_NYTT_ARBETE.getId(),
+                                ArbetslivsinriktadeAtgarderVal.BESOK_PA_ARBETSPLATSEN.getId(),
+                                ArbetslivsinriktadeAtgarderVal.ERGONOMISK_BEDOMNING.getId(),
+                                ArbetslivsinriktadeAtgarderVal.HJALPMEDEL.getId(),
+                                ArbetslivsinriktadeAtgarderVal.KONFLIKTHANTERING.getId(),
+                                ArbetslivsinriktadeAtgarderVal.KONTAKT_MED_FORETAGSHALSOVARD.getId(),
+                                ArbetslivsinriktadeAtgarderVal.OMFORDELNING_AV_ARBETSUPPGIFTER.getId(),
+                                ArbetslivsinriktadeAtgarderVal.OVRIGT.getId()
+                            )
+                        )
+                        .build(),
+                    CertificateDataValidationHide.builder()
+                        .questionId(AVSTANGNING_SMITTSKYDD_SVAR_ID_27)
+                        .expression(singleExpression(AVSTANGNING_SMITTSKYDD_SVAR_JSON_ID_27))
+                        .build()
+                }
+            )
+            .build();
+    }
+
+    private static List<CertificateDataValueCode> createAtgarderCodeList(LisjpUtlatandeV1 internalCertificate) {
+        if (internalCertificate.getArbetslivsinriktadeAtgarder() == null) {
```

Comment:
Consider just passing arbetslivsinriktadeAtgarder as the argument instead of the whole certificate.